scripts/capture-scrutinize.sh

```diff
@@ -128,6 +128,9 @@ function scrutinizeForAdmintools() {
     OP_FILE="$ns.$v.scrutinize.tar"
     logInfo "Running scrutinize"
     set -o xtrace
+    # scrutinize needs to be able to ssh to the pod
+    POD_IP=$(kubectl get pod -n $ns $pod -o jsonpath='{.status.podIP}')
+    kubectl exec -t -n $ns $pod -- bash -c "ssh-keyscan $POD_IP >> ~/.ssh/known_hosts"
```

Comment:
Why do we need this now?

Reply:
Scrutinize still has a bug. See my comment under this Jira: https://jira.verticacorp.com/jira/browse/VER-96447. This is a workaround before client team fixed the scrutinize.